src/frontend/efiling-frontend/src/domain/package/package-review/PackageReview.js

```diff
@@ -114,10 +114,12 @@ export default function PackageReview({ page: { header, packageId } }) {
     });
   }
 
-  function handleKeyDown() {
-    getSubmissionSheet(packageId).catch((err) => {
-      errorRedirect(sessionStorage.getItem("errorUrl"), err);
-    });
+  function handleKeyDown(e) {
+    if (e && e.keyCode === 13) {
+      getSubmissionSheet(packageId).catch((err) => {
```

Comment:
same here for the name of the method `saveSubmissonSheetLocal` ?

Reply:
Done. Refactored.